specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/EndpointCertificates.json

```diff
@@ -0,0 +1,296 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved an endpoint certificate.",
+            "schema": {
+              "$ref": "#/definitions/EndpointCertificate"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 ResourceNotFound - The requested resource was not found.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription."
+          }
+        },
+        "x-ms-examples": {
+          "Gets an endpoint certificate.": {
+            "$ref": "./examples/EndpointCertificatesGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/endpointCertificates": {
+      "get": {
+        "tags": [
+          "EndpointCertificates"
+        ],
+        "description": "List certificates used on endpoints on the target instance.",
+        "operationId": "EndpointCertificates_ListByInstance",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a list of endpoint certificates.",
+            "schema": {
+              "$ref": "#/definitions/EndpointCertificateListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get a list of endpoint certificates.": {
+            "$ref": "./examples/EndpointCertificatesListByInstance.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EndpointCertificateProperties": {
+      "description": "The properties of an endpoint certificate.",
+      "type": "object",
+      "properties": {
+        "publicBlob": {
+          "description": "The certificate public blob",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
```

Comment:
>Resource [](http://example.com/codeflow?start=5&length=8)

Better to use common types definitions for this and ProxyResource definition below. Not blocking from ARM perspective, but definitely a good idea.